src/commands/build/index.js

```diff
@@ -24,26 +23,15 @@ class BuildCommand extends Command {
   }
 
   // Retrieve Netlify Build options
-  async getOptions() {
-    const { site } = this.netlify
+  getOptions() {
+    const {
+      site: { id: siteId }
+    } = this.netlify
     const {
       flags: { dry = false, context }
     } = this.parse(BuildCommand)
     const [token] = this.getConfigToken()
-
-    // Try current directory first, then site root
-    const config = (await getConfigPath()) || (await getConfigPath(undefined, this.netlify.site.root))
```

Comment:
This line is now done under the hood by `@netlify/config` (through `@netlify/build`) when the `config` option is `undefined`.